addons/event_sale/models/event.py

```diff
@@ -60,6 +74,32 @@ def _is_event_registrable(self):
         else:
             return False
 
+    @api.depends('event_ticket_ids.start_sale_date')
+    def _compute_start_sale_date(self):
+        for event in self:
+            start_dates = [ticket.start_sale_date for ticket in event.event_ticket_ids if ticket.start_sale_date]
+            event.start_sale_date = min(start_dates) if start_dates else False
+
+    @api.depends('sale_order_lines_ids')
+    def _compute_sales_total_price(self):
+        for event in self:
+            event.sales_total_price = sum([
+                event.currency_id._convert(
+                    sale_order_line_id.price_reduce_taxexcl,
+                    sale_order_line_id.currency_id,
+                    sale_order_line_id.company_id,
+                    sale_order_line_id.order_id.date_order)
+                for sale_order_line_id in event.sale_order_lines_ids
+            ])
+
+    def _compute_event_tickets_available(self):
```

Comment:
I would split and mode this field definition. Indeed I think we should have it available directly on event model to be able to use it. Morever I found _is_event_registrable method that I think could be merged with yours (it does not seem used anymore but I think it was interesting)

  * base, in event: based on event date_end  and seats;
  * event_sale: tickets availability + check that product is published (_is_event_registrable)